paasta_tools/autoscaling/autoscaling_service_lib.py

```diff
@@ -725,6 +716,13 @@ def autoscale_marathon_instance(
                         ),
                         level="event",
                     )
+            else:
+                write_to_log(
```

Comment:
I still don't think we should log this.
We don't need to continuously log for every service/instance that the autoscaler isn't doing anything, even at the "debug" level for paasta logs

Reply:
I really don't think it's very costly, and would be helpful when reading logs to debug autoscaler issues.